NineAnimator/Models/Anime Source/animehub.ac/AnimeHub+Episode(OLD).swift

```diff
@@ -0,0 +1,122 @@
+//
+//  This file is part of the NineAnimator project.
+//
+//  Copyright © 2018-2020 Marcus Zhou. All rights reserved.
+//
+//  NineAnimator is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  NineAnimator is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with NineAnimator.  If not, see <http://www.gnu.org/licenses/>.
+//
+
+// This Version of AnimeHub+Episode has extra checks to make sure
+// that the user selected server is available. It also provides enhanced
+// error messages (EpisodeServerNotAvailableError).
+// However, since it makes multiple network requests, it is very slow.
+// Therefore we are disabling it.
+
+import Foundation
+import SwiftSoup
+
+extension NASourceAnimeHub {
+    static let knownServers = [
+        "fserver": "Fserver",
+        "fdserver": "FDserver",
+        //"xserver": "Xserver", Seems like this no episodes support this server
+        "oserver": "Oserver",
+        "mpserver": "MPserver",
+        "yuserver": "YUserver"
+        //"hserver": "Hserver" Excluding until we update HydraX parser
+    ]
+    /// Represents the response from AnimeHub episode endpoint
+    struct EpisodeResponse: Decodable {
```

Comment:
If `struct EpisodeResponse` is not used anywhere else, you should mark it as `private` or `fileprivate`.